website/content/terminal/quickstart/installation.md

```diff
@@ -245,7 +245,7 @@ We can use IPC socket to connect Desktop.
 Add this setting to your `.env` file.
 
 ```bash
-OPENBB_BACKEND=Qt5Agg
```

Comment:
Tkinter (Tk) is the default backend on most linux distros. Instructions to use Qt as a matplotlib backend was contributed by a user of some rare linux distro. I suggest this setting goes into the troubleshooting section and not be present in the general documentation

Reply:
Removed from this section.